lib/matplotlib/pyplot.py

```diff
@@ -49,29 +49,29 @@
 import sys
 import threading
 import time
+from typing import cast, overload
 
 from cycler import cycler
 import matplotlib
 import matplotlib.colorbar
 import matplotlib.image
 from matplotlib import _api
-from matplotlib import rcsetup, style
+from matplotlib import rcParams as rcParams, style as style  # Re-exported for typing.
 from matplotlib import _pylab_helpers, interactive
 from matplotlib import cbook
 from matplotlib import _docstring
 from matplotlib.backend_bases import (
     FigureCanvasBase, FigureManagerBase, MouseButton)
 from matplotlib.figure import Figure, FigureBase, figaspect
 from matplotlib.gridspec import GridSpec, SubplotSpec
-from matplotlib import rcParams, rcParamsDefault, get_backend, rcParamsOrig
-from matplotlib.rcsetup import interactive_bk as _interactive_bk
+from matplotlib import rcsetup, rcParamsDefault, get_backend, rcParamsOrig
 from matplotlib.artist import Artist
 from matplotlib.axes import Axes, Subplot  # type: ignore
 from matplotlib.projections import PolarAxes  # type: ignore
 from matplotlib import mlab  # for detrend_none, window_hanning
 from matplotlib.scale import get_scale_names
 
-from matplotlib import cm
+from matplotlib import cm as cm
 from matplotlib.cm import _colormaps as colormaps
```

Comment:
This may be one that we wish to reexport, may have to be something like:
```python
from matplotlib.cm import _colormaps

colormaps = _colormaps
```

Or adding a `__all__` to pyplot

Imported values are only re-exported if it is `import foo as foo` where the names match (for the type checker, of course runtime `*` imports pick up everything if there is no `__all__`)

Same may be needed for `color_sequences`, perhaps

Also could see `get_backend` being rexported (though the `foo as foo` way would work for that)

Reply:
OK, that seems to have worked; added that assignment version.